src/test/run-make-fulldeps/compiler-rt-works-on-mingw/Makefile

```diff
@@ -1,17 +1,9 @@
 -include ../tools.mk
 
-ifneq (,$(findstring MINGW,$(UNAME)))
-ifndef IS_MSVC
+# only-mingw
```

Comment:
As we're using `parse_cfg_name_directive`, `mingw` isn't a recognized configuration. Perhaps try 

```makefile
# only-windows
# only-gnu
```

Reply:
Fixed.